tests/routes/projects.test.ts

```diff
@@ -26,6 +26,23 @@ describe('Test projects routes', () => {
     expect(data).toMatchSnapshot();
   });
 
+  it('should return documents with 0 assets', async () => {
+    const res = await request(app).get('/projects/irrelevant-docs/master/documents');
+    expect(res.status).toBe(200);
+    const data = res.text.split('\n');
+    expect(data).toMatchSnapshot();
+  });
+
+  it('should return documents with 0 pages', async () => {
+    const res = await request(app).get('/projects/no-pages/master/documents');
+    expect(res.status).toBe(200);
+    const data = res.text.split('\n');
+    // Data should only have 2 pieces of information: timestamp and metadata
+    expect(data).toHaveLength(2);
+    expect(JSON.parse(data[0])).toHaveProperty('type', 'timestamp');
+    expect(JSON.parse(data[1])).toHaveProperty('type', 'metadata');
+  });
+
   it('should return documents updated after given timestamp', async () => {
```

Comment:
minor. would add the test for the AssetStream to have the updated asset as well 👍

Reply:
As part of this PR, I added a [second asset](https://github.com/mongodb/snooty-data-api/pull/11/files#diff-7e4a29c8b0b233ad42773f4a8bfac104e422722ddde4a63a4d40471c0ae36de3R41-R45) to one of the updated pages and the snapshot shows that a second asset was not added. One of the assets are updated while the other is not.

Do you think it would be better to make this case be explicit? Have some sort of logic that checks for the number of assets in the response?

Reply:
i missed the snapshot diff 🙏 

https://github.com/mongodb/snooty-data-api/blob/1e34289f89ab0393d67bd937fb68130e44a655b8/tests/routes/__snapshots__/projects.test.ts.snap#L23

above shows the missing [asset mentioned above](https://github.com/mongodb/snooty-data-api/blob/1e34289f89ab0393d67bd937fb68130e44a655b8/tests/sampleData/updatedDocuments.ts#LL42C20-L42C84) 👍